README.md

```diff
@@ -89,6 +89,19 @@ git rebase develop
 git push origin my-pr-branch --force[-with-lease]
 ```
 
+We would also like to implement [Commit Message Standard](https://getpocket.atlassian.net/wiki/spaces/PE/pages/2817458212/Commit+Message+Standard#Should-we-have-a-standard-commit-strategy-across-engineer%3F) in PR comments.
```

Comment:
@timc-mozilla Given this repo is public, can you put the commit standard we chose in here? Check out web-teams pr template that we could utilize https://pocket.slack.com/archives/C03QVL4SU/p1667843851931579